sonic_db_config/db_config.go

```diff
@@ -168,7 +185,8 @@ func GetDbTcpAddr(db_name string, ns string) string {
 	return hostname + ":" + strconv.Itoa(port)
 }
 
-func DbGetNamespaceAndConfigFile(ns_to_cfgfile_map map[string]string) {
+func DbParseConfigFile(name_to_cfgfile_map map[string]string) {
```

Comment:
>DbParseConfigFile [](http://example.com/codeflow?start=5&length=17)

Let's explore using swss-common SonicDBConfig class instead of reimplementing in this file. We can raise another PR, and let this PR depending on that PR.